apps/client/src/features/app-settings/panel/general-panel/GeneralFooter.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Button } from '@chakra-ui/react';
+
+import style from './GeneralPanel.module.scss';
+
+interface GeneralFooterProps {
+  formId: string;
+  handleRevert: () => void;
+  isDirty: boolean;
+  isValid: boolean;
+  isSubmitting: boolean;
+}
+
+export default function GeneralFooter(props: GeneralFooterProps) {
```

Comment:
I dont think we need this footer separation anymore
This used to happen as we attempted on reusing the footer elements across tabs of the modal.

We no longer have this structure, this buttons can be in the form, which makes everything much easier (even if more repetitive)